docs/MigrationGuide.md

```diff
@@ -57,6 +57,14 @@ val avroFixedSourceFactory = FixedKafkaAvroSourceFactory[GenericData.Record](pro
 
 Also, path to classes AvroSourceFactory was changed from avro.KafkaAvroSourceFactory to avro.source.* 
 
+* [#981](https://github.com/TouK/nussknacker/pull/981) Refactor Kafka and KafkaAvro serialization package
```

Comment:
The same as above. Why you want to force user to get to know your path of achieving the final target? He/she deployed 0.1.2 and want to know brief info what to change in his/her avro usage.

Reply:
I created only one point with information about PRs.